content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -290,13 +290,18 @@ parameters:
   type: io1
   iopsPerGB: "50"
   encrypted: "true"
+  tagSpecification_1: "key1=value1"
+  tagSpecification_2: "key2=value2"
 allowedTopologies:
 - matchLabelExpressions:
   - key: topology.ebs.csi.aws.com/zone
     values:
     - us-east-2c
 ```
 
+`tagSpecification`: Tags with this prefix are applied to dynamically provisioned EBS volumes.
+For more details, refer to [AWS_EBS_CSI_Driver StorageClass Tagging](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/tagging.md).
```

Comment:
@sftim 
AWS provide the EBS CSI Driver as an add-on for EKS, and for manual setups, they refer to the same source code repository for guidance.

Since there's no official AWS documentation available for this specific feature, the repo link is the most accurate source of information currently.

Reply:
There's https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html

We can encourage AWS to document it there, right? If we ask around there may well be AWS staff who can help.

(I know that AWS link to the GitHub pages for readers to follow; as SIG Docs, we try to encourage cloud providers to make even more complete documentation).

Reply:
@sftim 
I agree that it would be great to have this feature documented in AWS's official user guide. However, if we look into the storageclass.md page, we can see that source code repo links with branch referencing are used in every other example.

Given that pattern, it seemed appropriate to follow the same approach for consistency until AWS provides official documentation.

That said, I'm still happy to encourage AWS to document it more thoroughly in their user guide.

Reply:
We should fix all the other places too, eventually.

Reply:
I think this is a long-term optimization job, maybe it's better to track it in one issue?

Reply:
@sftim 
I understand your concern. If referencing the source code repo is considered a bad practice or convention, we can simply omit the documentation link entirely.

The explanation of `tagSpecification` provides sufficient understanding, and users can find further documentation themselves if needed.

Reply:
Omitting the link to the source code sounds good to me.

Folks at AWS: we'd love to have **you** document this and as the associated code is in-project, we can totally link to your docs for additional detail.

Reply:
Hi @sftim,
I’d really appreciate it if you could take a moment to review the changes you proposed so we can finally move forward.

Thanks.